app/views/components/answerDisplay.html

```diff
@@ -4,6 +4,7 @@
   <span ng-switch-when="addressQuestion">{{answer.value.address1}} {{answer.value.address2}}<br>{{answer.value.city}}, {{answer.value.state}} {{answer.value.zip}}</span>
   <span ng-switch-when="textareaQuestion" class="eventhub-linebreak">{{answer.value}}</span>
   <span ng-switch-when="phoneQuestion">{{answer.value | tel}}</span>
+  <span ng-switch-when="dateQuestion">{{answer.value}}</span>
```

Comment:
This line is not needed if no formatting or manipulation is going to take place.